db/migrate/20161117004319_create_revisions.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :revisions do |t|
       t.references :user, foreign_key: true
       t.references :article, foreign_key: true
+      t.integer :original_article_id
     end
```

Comment:
New migrations for things going forward so we don't repeat the issues I created with author_id